diem-unified-charter.md

```diff
@@ -1,9 +1,6 @@
 # Introduction
 
 An emblem is a device, symbol, or figure adopted and used as an identifying mark.
-As such, emblems provide a mechanism for conveying a set of attributes bound to an asset. 
-An emblem may be presented to a validator and attributes are asserted by an issuer.
```

Comment:
Too much detail in introduction, we  introduce roles in the architecture considerations section.
I also feel this is somewhat redundant given the current text

Reply:
Yes, I also felt this doesn't belong here but in the architecture section. However, I don't think you moved the text there, did you? That fine, but I wanted to point out that we are now loosing the bit on "attributes". However, maybe this is not important or actually too detailed for the charter, anyway?